templates/base.html

```diff
@@ -0,0 +1,13 @@
+<head>
+  {% block title %}
+    <title>Taxi Service</title>
+  {% endblock %}
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+  {% block sidebar %}
+    {% include "includes/sidebar.html" %}
+  {% endblock %}
+  {% block content %}{% endblock %}
```

Comment:
it seems to me, that sidebar should be placed at the left/right side of a page, not at the same level as content ))